waiter/integration/waiter/kubernetes_scheduler_integration_test.clj

```diff
@@ -134,7 +134,7 @@
 
           ;; Test that the killed instances' logs were persisted to S3.
           ;; This portion of the test logic was modified from the active-instances tests above.
-          (let [log-bucket-url (k8s-log-bucket-url waiter-url)
+          (let [log-bucket-url (-> (default-scheduler-settings waiter-url) :log-bucket-url)
```

Comment:
This only works if the default scheduler is kubernetes. Instead the function should retrieve kubernetes scheduler settings and then lookup `log-bucket-url`

Reply:
this is only called when the scheduler is k8s

Reply:
updated